aries/aries_vcx_core/src/anoncreds/credx_anoncreds.rs

```diff
@@ -200,23 +115,19 @@ impl IndyCredxAnonCreds {
     where
         T: DeserializeOwned,
     {
-        let str_record = wallet.get_wallet_record_value(category, id).await?;
-        serde_json::from_str(&str_record).map_err(From::from)
+        let str_record = wallet.get_record(id, category).await?;
+        serde_json::from_str(str_record.get_value()).map_err(From::from)
```

Comment:
Should been noted in previous PR, but I just noticed it now. 

Looking at `did_doc` and `public_key` crates as reference, we tend to not use `get_` prefix on getter functions. Eg. example from `public_key`
```
pub fn key_type(&self) -> &KeyType {
        &self.key_type
    }

    pub fn key(&self) -> &[u8] {
        self.key.as_ref()
    }

```
example from `did_doc`
```
pub fn controller(&self) -> Option<&OneOrList<Did>> {
        self.controller.as_ref()
    }

    pub fn verification_method(&self) -> &[VerificationMethod] {
        self.verification_method.as_ref()
    }
```

I am actually fine keeping get on the BaseWallet method, because those are rather IO operations than just plain in memory "getters". But I would suggest `value` rather than `get_value` for the `Record` type (and analogically for its other getters)